packages/app/components/Queue/Student/StudentBanner.tsx

```diff
@@ -54,7 +54,7 @@ export default function StudentBanner({
   leaveQueue,
 }: StudentBannerProps): ReactElement {
   const { studentQuestion, studentQuestionIndex } = useStudentQuestion(queueId);
-  const isQueueOnline = useQueue(queueId).queue?.room === "Online";
+  const isQueueOnline = useQueue(queueId).queue?.room.startsWith("Online");
```

Comment:
did not abstract this file cause i think `isQueueOnline` is being used to show/hide some component